_topic_map.yml

```diff
@@ -305,8 +305,13 @@ Topics:
   File: understanding-networking
 - Name: Multiple networks
   File: managing_multinetworking
-- Name: Ingress controller configuration
-  File: ingress-controller-configuration
+- Name: Ingress
+  Dir: ingress
+  Topics:
+  - Name: Ingress Operator in OpenShift Container Platform
+    File: configuring-ingress-operator
```

Comment:
Why is the same filename referenced by both topic names?

Reply:
Wonky rebase. Fixing, thanks!

Reply:
This is referencing a file at `networking/configuring-ingress-operator.adoc` but the actual files are nested under `networking/ingress` which seems to break all the includes and cause the topic to be excluded from the build.

Reply:
Does the subdirectory include symlinks? For example, `installing/installing_aws/` symlinks to `../modules` and `../images`:

```
s -1 installing/installing_aws/
modules
images
installing-aws-account.adoc
installing-aws-customizations.adoc
installing-aws-default.adoc
uninstalling-cluster-aws.adoc
```

Reply:
No. Offtopic, but we really should find a way to kill the symlink pattern. It's actively hostile to tooling which would greatly improve the experience of iterating on docs.

Reply:
Thanks for pointing that out, we can look into the symlink pattern for sure. For now the file structure is set up correctly for asciibinder (at least I'm able to build it successfully). I'm going to go ahead and merge the PR to get it in for review, but we can and certainly will continue to iterate on it.